client/coral-admin/src/components/UserDetail.js

```diff
@@ -82,13 +63,19 @@ export default class UserDetail extends React.Component {
     }
   }
 
-  showAll = () => {
-    this.props.changeStatus('all');
+  changeTab = (tab) => {
+    this.props.changeStatus(tab);
```

Comment:
We should change the prop name from `changeStatus` to `changeTab`, also related props in the container and redux actions to make it more clear.